src/index.js

```diff
@@ -0,0 +1,273 @@
+/* eslint no-use-before-define: 0, no-extra-parens: 0 */
```

Comment:
contrived attempt at an example. Amongst helper functions it isn’t always possible to have everything defined “first’. `no-use-before-define` would matter in a synchronous calling environment (TDZ and all). I will re-enable this and reorg it back into order.

```js
const helper1 = () => helper3();
const helper2 = () => helper1();
const helper3 = () => helper2();
```